src/test/java/com/iexec/sms/tee/session/scone/SconeSessionMakerServiceTests.java

```diff
@@ -207,11 +185,11 @@ void setup() throws TeeSessionGenerationException {
 
         @Test
         void shouldGenerateMaaSession() throws Exception {
-            SconeSession actualCasSession = palaemonSessionService.generateSession(request);
+            final SconeSession actualCasSession = palaemonSessionService.generateSession(request);
```

Comment:
I asked to Scontain about the palaemon name. They answered me this is kind of an obsolete project name that has no more meaning. Can we start to replace this name with a more agnostic name representing the delivered service ?
```suggestion
            final SconeSession actualCasSession = sconeSessionMakerService.generateSession(request);
```